readme.md

```diff
@@ -63,10 +63,11 @@ Run the following commands:
 git clone git@github.com:wagtail/bakerydemo.git
 cd bakerydemo
 docker-compose up --build -d
+docker-compose run app /venv/bin/python manage.py createsuperuser #Follow the instructions, name your user `admin`
```

Comment:
Lemme know if this seems vague, or is even necessary.
Needed to run this before I could load the test data.